lib/python/qmk/cli/doctor/check.py

```diff
@@ -53,7 +70,22 @@ def _check_avr_gcc_version():
     version_number = ESSENTIAL_BINARIES['avr-gcc']['output'].strip()
     cli.log.info('Found avr-gcc version %s', version_number)
 
-    return CheckStatus.OK
+    # Right now all known arm versions are ok, so check that it can produce binaries
```

Comment:
Shouldn't this be: 

```suggestion
    # Right now all known avr versions are ok, so check that it can produce binaries
```

?